### PR TITLE
Create initial smoke test page

### PR DESCRIPTION
**Pull request by mistyhacks**
Initial smoke test page

/assign @chenopis

### CODE REVIEW

File: content/en/docs/test.md [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+
+This is in an H3 section.
+
+#### H4
+
+This is in an H4 section.
+
+##### H5
+
+This is in an H4 section.
```
Comment (zacharysarah) [inline content/en/docs/test.md]: H5 section

File: content/en/docs/test.md [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+
+###### H6
+
+This is in an H4 section.
```
Comment (zacharysarah) [inline content/en/docs/test.md]: H6 section

**Comment by mistyhacks** [comment]
@zacharysarah thanks, fixed.

**Comment by mistyhacks** [comment]
Keep in mind that this is not ready to merge. I can't actually figure out where the page renders / shows up in the Netlify output. Not sure if I am doing something wrong and it isn't being rendered as HTML at all.

**Comment by jaredbhatti** [comment]
I can see the page just fine here: 
https://deploy-preview-8349--kubernetes-io-master-staging.netlify.com/docs/test/

File: content/en/docs/test.md [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+| Heading cell 1 | Heading cell 2 |
+|----------------|----------------|
+| Body cell 1    | Body cell 2    |
+
+The header is optional. Any text separated by `|` will render as a table.
+
+Markdown tables have a hard time with block-level elements within cells, such as
+list items, code blocks, or multiple paragraphs. For complex or very wide
+tables, use HTML instead.
+
```
Comment (jaredbhatti) [inline content/en/docs/test.md]: This table doesn't seem to render correctly, but I can't see why: 
https://deploy-preview-8349--kubernetes-io-master-staging.netlify.com/docs/test/#tables

**Comment by jaredbhatti** [comment]
I see an issue with the

**Comment by mistyhacks** [comment]
@jaredbhatti The HTML table is really weird. It seems to be getting an extra (empty) `<th></th>` pair, but I'm not sure why. I'm pretty sure it's being added by Javascript, because it's not there when I view the source but it is there when I inspect the rendered page. I'm baffled that it's affecting the HTML table and not the Markdown table above it.

**Comment by mistyhacks** [comment]
I think the non-shortcode sidebar I added shows another bug, which I also noted in #8357.

File: content/en/docs/test.md [hunk 4]
```diff
@@ -0,0 +1,248 @@
...
+
+## Images
+
+To format an image, use similar syntax to [links](#links), but add a leading `!`
+character. The square brackets contain the image's alt text. Try to always use
+alt text so that people using screen readers can get some benefit from the
+image.
+
+![pencil icon](static/images/pencil.png)
```
Comment (chenopis) [inline content/en/docs/test.md]: None of these pencil icons seem to be showing up. I think `/static/*` needs to be moved into `/content/en/`.

**Comment by steveperry-53** [comment]
@mistyhacks, Thanks for doing this. One comment: In our current style, we don't include the $ prompt when we show commands.

File: content/en/docs/test.md (hunk 4 above)
Comment (mistyhacks) [reply content/en/docs/test.md]: I'm not sure what I need to do in this PR about the images...

**Comment by mistyhacks** [comment]
Thanks @steveperry-53 , I removed the `$` from the command prompt examples.

**Pull request merged**